canonicalwebteam/discourse/models.py

```diff
@@ -31,6 +31,22 @@ def get_topic(self, topic_id):
 
         return response.json()
 
+    def get_topics(self, topic_ids):
+        topics = ",".join([str(i) for i in topic_ids])
+
+        response = self.session.post(
+            "http://localhost/admin/plugins/explorer/queries/2/run",
```

Comment:
Replace localhost with actual host of the discourse.

Reply:
@tbille Ups, I did some changes to improve this part, please review it again.